include/ChaiVM/interpreter/code-manager/klass.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <vector>
+
+#include "ChaiVM/interpreter/instruction.hpp"
+#include "ChaiVM/memory/linear-allocator.hpp"
+#include "ChaiVM/memory/linear-buffer.hpp"
+#include "ChaiVM/types.hpp"
+
+namespace chai::interpreter {
+
+union Data {
+    int64_t ch_int_;
+    double ch_float_;
+    chsize_t ch_ref_;
+};
```

Comment:
@levBagryansky why do we need this union?

Reply:
@c71n93 Removed